src/app/form/form.component.html

```diff
@@ -1,3 +1,193 @@
-<p>form works!</p>
+<div class="wrapper">
+    <div class="form-wrapper">
+        <div class="logo-wrapper">
+            <p>Logo Here</p>
+        </div>
+        <div class="info-wrapper">
+            <p>
+                Tell us about you!
+            </p>
+            <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Tellus orci ac auctor augue mauris augue neque gravida.
+            </p>
+        </div>
+        <div class="form-step-wrapper">
+            <mat-stepper #stepper [orientation]="orientationType" linear="false">
+                <mat-step [stepControl]="contactInfoFormGroup" errorMessage="Form not complete">
+                    <ng-template matStepLabel class="label">Contact Information</ng-template>
+                    <form [formGroup]="contactInfoFormGroup">
```

Comment:
we could probably abstract the each form into components at some point. Not now.